cypress/e2e/app_commands/db_state.rb

```diff
@@ -0,0 +1,20 @@
+logger.debug "running db_state #{command_options}"
+option = command_options.to_s.downcase
+case option
+when "capture"
+  DatabaseCleaner.start
+  puts "DatabaseCleaner started"
+when "restore"
+  DatabaseCleaner.clean
+  CypressOnRails::SmartFactoryWrapper.reload
+
+  if defined?(VCR)
+    VCR.eject_cassette # make sure we no cassette inserted before the next test starts
+    VCR.turn_off!
+    WebMock.disable! if defined?(WebMock)
```

Comment:
Note, I was going to put this into a class/module but it's evaluated in the context of the cypress on rails command executor so finding/prepending classes was more effort than needed.  If we want to move the logic into methods, we can do that later.